lib/msal-common/src/cache/CacheManager.ts

```diff
@@ -803,8 +803,8 @@ export abstract class CacheManager implements ICacheManager {
         const entityScopeSet: ScopeSet = ScopeSet.fromString(entity.target);
         const requestTargetScopeSet: ScopeSet = ScopeSet.fromString(target);
 
-        if (!requestTargetScopeSet.containsOnlyDefaultScopes()) {
-            requestTargetScopeSet.removeDefaultScopes(); // ignore default scopes
+        if (!requestTargetScopeSet.containsOnlyOIDCScopes()) {
+            requestTargetScopeSet.removeOIDCScopes(); // ignore OIDC scopes
```

Comment:
I'm hoping that the containsOnly part will still support userinfo tokens, which are tokens requested with scope=openid. Will this cache those tokens, issued with scope=openid [profile email]

Reply:
Nevermind, I see it in the tests below, seems like.

Reply:
Yes, this will only remove OIDC scopes if they are included with non-OIDC scopes.